integration-tests/mysql_test.go

```diff
@@ -192,6 +191,19 @@ var _ = Describe("Mysql", func() {
 				Entry("update authorized_network_id", map[string]any{"authorized_network_id": "another-authorized-network_id"}),
 			)
 		})
+
+		Context("should allow users update parameters", func() {
+
+			DescribeTable("not marked as prohibited to update not included in the plan",
```

Comment:
Maybe "not marked as prohibit_update and not specified in the plan"?

Reply:
I would actually eliminate the `Context` blocks and just leave the DescribeTables. Would probably have three:
1.  "Can update properties not flag as prohibit_update and not specified in the plan"
2.  "Cannot update properties flag as prohibit_update"
3.  "Cannot update properties that are specified in the plan"

I would also make sure that the tests for 2 is done with a plan that doesn't specify any of the prohibit_update properties and 3 with a plan that specifies properties not flagged as prohibit_update. Just to make sure the tests pass/fail for the intended reasons.

Reply:
That sounds fantastic. I'll implement it. Thank you very much!